encord/user_client.py

```diff
@@ -296,6 +296,7 @@ def create_project(
         project_description: str = "",
         ontology_hash: str = "",
         workflow_settings: ProjectWorkflowSettings = ManualReviewWorkflowSettings(),
+        workflow_template_hash: Optional[str] = None,
```

Comment:
So that directly clashes with the `workflow_settings`... in the sense that the QA workflow settings would make no sense with a template hash. What should we do about this? 
The least is documentation, maybe we can make it nice and ensure from the type system that the clash is not possible

Reply:
I don't think we can 100% solve this technically, it will still be confusing, especially because of the name clash.
So I think the best we can do is to explain what is going on in the docs as a short term solution, and start deprecating pre-workflow functionality as soon as possible to remove the confusion.

Reply:
That's fair enough. Let's have a clear error message then from the API server to make sure that clients who do not read the docs too much will not get frustrated